schema/release-schema.json

```
@@ -1736,7 +1736,274 @@
             "string",
             "null"
           ],
-          "description": "The country name. For example, United States."
+          "description": "The country name. For example, United States.",
+          "deprecated": {
+            "description": "This field is deprecated, because OCDS isn't the place to annotate a country with additional information like its human-readable name. ISO3166-1 alpha-2 country codes can be provided in `.country`.",
+            "deprecatedVersion": "1.2"
+          }
+        },
+        "country": {
+          "title": "Country code",
+          "description": "The country where the address is located, from the closed [country](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#language) codelist. If you need to use a user-assigned code from ISO 3166-1 alpha-2, [create an issue](https://github.com/open-contracting/standard/issues) to discuss its addition to the country codelist.",
```

Reworded first part per my earlier comment. Reworded second sentence to more clearly describe the reason for creating an issue.

```suggestion
          "description": "The country, from the closed [country](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#language) codelist. If a country is missing, [create an issue](https://github.com/open-contracting/standard/issues) to discuss the addition of a user-assigned code to the country codelist.",
```